index.html

```diff
@@ -11,15 +11,18 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
     <!-- Material Icons -->
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <!-- Progress bar -->
+    <script src="js/progressbar.min.js"></script>
 
```

Comment:
To implement the progress bar, you can use a library like [progressbar.js](https://kimmobrunfeldt.github.io/progressbar.js/). First, add the following script tag to the <head> section: